sbtx

```diff
@@ -34,8 +34,8 @@
 
 set -o pipefail
 
-declare -r sbt_release_version="1.4.4"
-declare -r sbt_unreleased_version="1.4.4"
+declare -r sbt_release_version="1.4.6"
```

Comment:
Should it go to dependency updates?

Reply:
Added short mention of update. 
Unfortunately, the [wrapped](https://github.com/paulp/sbt-extras) is not versioned.

Reply:
Also updated the changes file to the agreed format.